lib/app.js

```diff
@@ -1,17 +1,24 @@
 const express = require('express');
 const cookieParser = require('cookie-parser');
-
-const authenticate = require('./middleware/authenticate');
-
+const cors = require('cors');
 const app = express();
 
 // Built in middleware
 app.use(express.json());
 app.use(cookieParser());
+app.use(
+  cors({
+    origin: [
+      'https://alchemy-shopping-front-end-demo.netlify.app',
+      'http://localhost:5501',
+    ],
+    credentials: true,
+  })
+);
 
 // App routes
-
 app.use('/api/v1/users', require('./controllers/users'));
+app.use('/api/v1/items', require('./controllers/items'));
```

Comment:
since all the routes use `authenticate` you can also add that middleware here
`app.use('/api/v1/items', authenticate, require('./controllers/items'));`